src/components/layout/Sidebar.js

```diff
@@ -112,6 +115,10 @@ class Sidebar extends Component {
     this.setState({ tooltipEnabled: false });
   }
 
+  collapseMenu() {
+    this.setState({ isCollapsed: !this.state.isCollapsed });
```

Comment:
Hi @SpecialAro can we change this line to something like this
`this.setState(({ isCollapsed }) => ({ isCollapsed: !isCollapsed }));`

Reply:
Will implement this on next commit! Thank you :D